index.html

```diff
@@ -12,10 +12,13 @@
     <title>mobileFirst</title>
 </head>
 <body>
+    <div class="header">
+        <a class="logo" href="">My Logo</a>
     <nav>
-        <header>My Logo</header>
-        <img src="Images/menu-button.png" alt="">  
+        <img class="nav" src="Images/menu-button.png" alt="">  
     </nav>
+    </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please change `<div class="header">`❌ tag to `<header class="header">`✔️ because all previous reviewers mention like below:

```html
<header>

    <a class="logo" href="">My Logo</a>

    <nav>
        <img src="Images/menu-button.png" alt="">  
        <img class="nav" src="Images/menu-button.png" alt="">  
    </nav>

</header>
```